build.sbt

```diff
@@ -14,7 +14,7 @@ val akkaVersion = if (Nightly) akkaVersion26 else "2.5.30"
 val AkkaBinaryVersion25 = "2.5"
 val AkkaBinaryVersion26 = "2.6"
 val AkkaBinaryVersion = if (Nightly) AkkaBinaryVersion26 else AkkaBinaryVersion25
-val kafkaVersion = "2.4.1"
+val kafkaVersion = "2.4.1.1"
 val embeddedKafkaVersion = kafkaVersion
```

Comment:
But this doesn't look correct...

Reply:
There is indeed a 2.4.1.1 of `embedded-kafka`, but we use this variable to assign both Embedded Kafka and Apache Kafka.  Apache Kafka does not have a 2.4.1.

I think Embedded Kafka 2.4.1.1 was released with some bugfixes that we've already worked around, but all I have to go on is the commit log since there aren't any release notes (https://github.com/embeddedkafka/embedded-kafka/releases/tag/v2.4.1.1).